From cc750d67acc299dc26a0691883c7d8a5983a3cdc Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 7 Dec 2012 17:34:21 +0100
Subject: [PATCH 170/204] umount: fix check for DM changed

delete_dm() returns 0 on success. The check for "r > 0" was likely
a copy&paste error from the loopback code where "r > 0" makes sense.
(cherry picked from commit c678406681d32d56730b9e9c002d5500d7aa7f8b)
---
 src/core/umount.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/umount.c b/src/core/umount.c
index 5989a4c..408e1cf 100644
--- a/src/core/umount.c
+++ b/src/core/umount.c
@@ -536,7 +536,7 @@ static int dm_points_list_detach(MountPoint **head, bool *changed) {
 
                 if ((r = delete_dm(m->devnum)) >= 0) {
 
-                        if (r > 0 && changed)
+                        if (changed)
                                 *changed = true;
 
                         mount_point_free(head, m);
-- 
1.8.1.5

